@shared/api/internal.ts

```diff
@@ -193,31 +199,41 @@ export const getAccountDetails = async (
     networkPassphrase: NETWORK_PASSPHRASE,
   });
 
-  let payments = null;
   let balances = null;
   let isFunded = null;
+  let operations = [] as Array<HorizonOperation>;
 
   try {
     ({ balances } = await dataProvider.fetchAccountDetails());
-    const transactionData = await dataProvider.fetchPayments({
-      limit: TRANSACTIONS_LIMIT,
-    });
-    payments = transactionData?.records || null;
+    isFunded = true;
```

Comment:
evidently `isAccountFunded ` just calls `fetchAccountDetails`and returns `false` if that call fails. So, instead of doing 2 networks requests, i'll do that myself here and get both on 1 request